firestore/testapp/src/common_main.cc

```diff
@@ -25,14 +25,14 @@
 #include "firebase/util.h"
 
 // Thin OS abstraction layer.
-#include "main.h"  // NOLINT
+#include "main.h" // NOLINT
 
 const int kTimeoutMs = 5000;
 const int kSleepMs = 100;
 
 // Waits for a Future to be completed and returns whether the future has
 // completed successfully. If the Future returns an error, it will be logged.
-bool Await(const firebase::FutureBase& future, const char* name) {
+bool Await(const firebase::FutureBase &future, const char *name) {
```

Comment:
We are attaching asterisks and ampersands to the type name, not the variable name in our code (including the public headers and the documentation), so we should follow the same style here. `clang-format` is likely configured to use a heuristic to apply the style prevalent in the file.

Is there a `.clang-format` file in the repo? If not, you might want to add one (just copy our existing one from the [C++ repo](https://github.com/firebase/firebase-cpp-sdk/blob/main/firestore/.clang-format)). The relevant settings are `DerivePointerAlignment` (whether to apply the heuristic, should be `false`) and `PointerAlignment` (should be `Left`).

Reply:
Done

Reply:
If you used a custom `.clang-format` file, it's probably a good idea to check it in.

Reply:
ah, I meant to do that and forgot. I'll send another PR for it.